libraries/botframework-config/src/botConfiguration.ts

```diff
@@ -106,7 +106,8 @@ export class BotConfiguration extends BotConfigurationBase {
      * @returns A Promise with the new BotConfiguration instance.
      */
     static async load(botpath: string, secret?: string): Promise<BotConfiguration> {
-        const json: string = await txtfile.read(botpath);
+        // eslint-disable-next-line security/detect-non-literal-fs-filename
+        const json: string = fs.readFileSync(botpath, 'utf8');
```

Comment:
Shouldn't this be replaced with an async function?

Reply:
It doesn't work, executes the other functions and later read
![image](https://github.com/southworks/botbuilder-js/assets/122501764/72a465c9-db12-4e53-a114-1c14c689af94)

Reply:
Check how they used it in botbuilder-dialogs-adaptive-runtime-integration-azure-functions\src\index.ts. They added a promise.

Reply:
resolved